### ISSUE TITLE
Generator only example

### ISSUE THREAD

**Issue by vonj** [lloyd/yajl#199]
Could we please have a generator only example.

The current example for creating a JSON file is converting a JSON file...

I find that confusing. Could we have a small C program which creates a simple JSON file without reading  in another?

**Comment by IQYottabit** [comment]
I agree with @vonj vonj, would be great to have a small example program which creates a simple JSON file, that would be great for the beginners.

**Comment by slinkinone** [comment]
See [this PR](https://github.com/lloyd/yajl/pull/114).
